src/test/java/uk/gov/companieshouse/officerfiling/api/controller/OfficerFilingControllerImplValidationIT.java

```diff
@@ -241,53 +241,6 @@ void createFilingWhenReferenceResignedOnInFutureThenResponse400() throws Excepti
                         is("must be a date in the past or in the present")));
     }
 
```

Comment:
Would it be possible to move all these integration tests into ValidationStatusControllerImplIT instead of deleting them? Testing in that class is quite light due to having these validation tests but if validation is no longer testable in this IT class then perhaps the tests should be moved rather than deleted.

Reply:
I didn't realise we hadn't added tests in the validation class, as the situation should be different, i.e. this was validating for a post, which is no longer valid,  but we should be validating for a validation_status call.
I'll check and see.

Reply:
I have now updated and added the test scenarios into the validation status class instead.